docs/dbaas/setting-up.md

```diff
@@ -93,7 +93,7 @@ alias kubectl='minikube kubectl --'
     ```sh
     eksctl create cluster --write-kubeconfig --name=your-cluster-name --zones=us-west-2a,us-west-2b --kubeconfig <PATH_TO_KUBECONFIG>
     ```
-2. Follow the instructions on [How to add a Kubernetes cluster](../../get-started/dbaas.md#add-a-kubernetes-cluster) with kubeconfig from the previous step.
+2. Follow the instructions on [How to add a Kubernetes cluster](../dbaas/get-started.md#add-a-kubernetes-cluster) with kubeconfig from the previous step.
 
     !!! note alert alert-primary ""
         If possible, the connection details will show the cluster's external IP (not possible with minikube).
```

Comment:
What do you mean "if possible"?

Reply:
I had the same question :D I'll rephrase this